pkg/settings/settings.go

```diff
@@ -77,3 +83,11 @@ func getEnvBool(name string, def bool) bool {
 
 	return boolean
 }
+
+func getClusterScopedBlacklist(name string, def []string) []string {
```

Comment:
A minor nit: since this is only used by the Migration subpackage, this should probably live in that file, or possibly just be inlined into Migration.Load()